### DISCUSSION TITLE
Unit test for pyrpl

### DISCUSSION THREAD

**Discussion by abregnsbo** [lneuhaus/pyrpl#487] [General]
How do I unit test pyrpl ?

There seem to have existed a unit test for pyrpl, mentioned in the github README.md, but the `nosetests` shell script is no longer present on the master branch. On the official documentation site the unit test section https://pyrpl.readthedocs.io/en/latest/developer_guide/unittests.html is empty.

**Comment by michaelcroquette** [comment]
We have now switched to pytest.
